scio-neo4j/src/main/scala/com/spotify/scio/neo4j/syntax/AllSyntax.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.neo4j.syntax
+
+trait AllSyntax extends ScioContextSyntax with SCollectionSyntax
```

Comment:
Is this required ? or can we directly make the `neo4j` package extend `ScioContextSyntax` and `SCollectionSyntax`

Reply:
To be honest I took it directly from `JdbcIo`, so I don't know what is the proper pattern in Scio.

Reply:
Ok! I'm not too familiar either with what is the best archetype for the modules :) We can leave it like this